build-system/pr-validation.yaml

```diff
@@ -48,10 +48,10 @@ jobs:
       - task: Npm@1
         inputs:
           command: "custom"
-          customCommand: "install -g markdownlint-cli"
+          customCommand: "install -g markdownlint-cli markdownlint-rule-titlecase"
       - task: CmdLine@2
         inputs:
-          script: 'markdownlint "docs/**/*.md"'
+          script: 'markdownlint "docs/**/*.md" --rules "markdownlint-rule-titlecase"'
```

Comment:
ℹ️ FYI, I opened a GitHub issue with a question to see if there's any way to move this info config. No way so far that I can see.

ℹ️ I also double-checked that this doesn't _only_ enable that rule (by locally breaking a different rule.)